wolfCLU/clu_src/hash/clu_hash.c

```diff
@@ -100,27 +100,27 @@ int wolfCLU_hash(char* in, char* out, char* alg, int size)
     }
 #endif
 #ifndef NO_SHA
-    else if (strcmp(alg, "sha") == 0) {
+    if (strcmp(alg, "sha") == 0) {
```

Comment:
This change was made because if NO_MD5 is defined then an "else if" would be left without a proceeding if statement. I do agree my solution is not the best and will be considering other viable solutions such as adding an initial if statement that always evaluates to false before checking available hashing functions.